modules/zero-trust-manager-oidc-config.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * security/zero_trust_workload_identity_manageer/zero-trust-manager-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="zero-trust-manager-oidc-config_{context}"]
+== Deploying and configuring the SPIRE OpenID Connect Discovery Provider
+
+You can configure the `SpireOIDCDiscoveryProviderConfig` Custom Resource (CR) to deploy and configure the SPIRE OpenID Connect (OIDC) Discovery Provider.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+* Run the following script to configure the `SpireOIDCDiscoveryProviderConfig`:
```

Comment:
@wgabor0427 In our docs, weT split this into 2 separate steps.
1. Creating the yaml file
2. Running oc apply i.e. `oc apply -f <filename.yaml>`